x-pack/plugins/observability/public/hooks/use_fetch_rules.ts

```diff
@@ -36,6 +36,7 @@ export function useFetchRules({ ruleLastResponseFilter, page, sort }: FetchRules
       const response = await loadRules({
         http,
         page,
+        searchText,
```

Comment:
@mgiota, the search is very strict. I.e. it shows only the exact matches results. Is it the intended behavior?

Reply:
Yep that's true. It looks like that the `name` field is saved as `keyword` at ES and that's why it works only for exact matches. This is a question I wanted to address with response-ops team. Current Stack Management > Rules pages has the same behavior. 

@XavierM Is there maybe an existing issue for this? Is it in purpose that search should work only for exact matches?